service/lint.ml

```diff
@@ -1,18 +1,26 @@
 open Current.Syntax
 module Docker = Conf.Builder_amd1
 
-let format_dockerfile ~base ~ocamlformat_version =
-  let open Dockerfile in
-  from (Docker.Image.hash base)
-  @@ run "opam depext ocamlformat=%s" ocamlformat_version
-  @@ run "opam install ocamlformat=%s" ocamlformat_version
-  @@ copy ~chown:"opam" ~src:["./"] ~dst:"./src" ()
-  @@ workdir "src"
-
-let v_from_opam ~ocamlformat_version ~base ~src =
+let v_fmt ~ocamlformat_source ~base ~src =
   let dockerfile =
-    let+ base = base and+ ocamlformat_version = ocamlformat_version in
-    format_dockerfile ~base ~ocamlformat_version
+    let open Dockerfile in
```

Comment:
Not sure why you chose to inline this function here? It was clearer to me as it was written before.

Reply:
Fixed.